homeassistant/components/comed_hourly_pricing/sensor.py

```diff
@@ -113,7 +113,7 @@ async def async_update(self):
                     url_string += "?type=currenthouraverage"
 
                 async with async_timeout.timeout(60):
-                    response = await self.websession.get(url_string)
+                    response = await self.websession.get(url_string, ssl=False)
```

Comment:
Disabling SSL checks is not a solution to issues like this, especially considering this is a public service. This PR basically introduces a security issue.

Reply:
@frenck From what I understand from [the documentation](https://docs.aiohttp.org/en/stable/client_advanced.html), this setting doesn't disable the checks, it only uses less strict checks, presumably more along the lines of what Chrome does in practice. The certificates currently pass under Chrome but not with the default SSL values from this library. 

> By default aiohttp uses strict checks for HTTPS protocol. Certification checks can be relaxed by setting ssl to False:
> 
> r = await session.get('https://example.com', ssl=False)

If I'm misunderstanding the source of the problem (that it's the strict checking that's causing the issue because of the missing intermediate certificate,) then this SO post may hold some clues. I'd play around but I don't yet know how to import a development version of an integration into HA (I've only been playing with HA for a couple of weeks.)

Reply:
> From what I understand from [the documentation](https://docs.aiohttp.org/en/stable/client_advanced.html), this setting doesn't disable the checks, it only uses less strict checks

That is correct, meaning aiohttp will accept anything. Even self-signed and invalid certificates.

Reply:
Actually `ssl=false` does appear to completely skip validation according to [this section of the documentation](https://docs.aiohttp.org/en/stable/client_reference.html).

Reply:
> does appear to completely skip validation according to

Yes. Which is the security issue :)

Reply:
So... what do we do when Home Assistant uses different rules for intermediate certificate downloads/checks than modern browsers? I agree disabling SSL checks is not the solution, but the SSL certificate is valid, the chain is just incomplete or is expected to be part of the OS instead of served via the remote webserver. So what is the solution here?

Reply:
Looks like they fixed it, it's working for me now suddenly with no changes required in HA. The entity just started responding.